### ISSUE TITLE
As a user I should be able view the alt text

### ISSUE THREAD

**Issue by thomasMinshull** [Code-Critique/xkcd-ios#8]
**Is your feature request related to a problem? Please describe.**
XKCD comics have alt text that contains additional funny content, that is not initially displayed to the user. The user should be able to access this additional content for each comic. 

**Describe the solution you'd like**
The user should be able to view this alt text after interacting with the comic. Long pressing on the comic could display the alt text, or alternatively, this could be triggered by shaking the device.
